docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :release:`1.1.0 <2020-06-18>`
```

Comment:
It's the first time we have a date in here. I'll just assume that is done on purpose and the formatting is right, since I'm not sure how to check.

Reply:
We had it before, [example](https://github.com/raiden-network/raiden/blame/develop/docs/changelog.rst#L45). [both styles work](https://raiden-network.readthedocs.io/en/latest/changelog.html)